src/Hero/Hero.component.tsx

```diff
@@ -0,0 +1,176 @@
+// REACT
+import * as React from 'react';
+
+// VENDOR
+import styled from 'styled-components';
+import classnames from 'classnames';
+// ANCHOR
+import {
+    Typography,
+    TypographyProps,
+} from '../Typography/Typography.component';
+import { Grid } from '..';
+import { colors } from '../theme';
+
+// Hero
+// ------------------------------------------------------------------------------------------------------------------
+
+interface StyledHeroProps {
+    background?: string;
+    color?: string;
+    align?: string;
+    minHeight?: string;
+}
+
+const StyledHero = styled.section<StyledHeroProps>`
+    background: ${props => props.background};
+    position: relative;
+    width: 100%;
+    margin: 0 auto;
+    display: block;
+    color: ${props => props.color};
+    text-align: ${props => props.align};
+    min-height: ${props => props.minHeight};
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+`;
+
+interface HeroProps {
+    background?: string;
+    align?: string;
+    color?: string;
+    className?: string;
+    children?: any;
+    minHeight?: string;
+    height?: string;
+}
+
+export const Hero = ({
+    background = 'transparent',
+    align = 'center',
+    color = colors.white.base,
+    minHeight = '7.5rem',
+    className,
+    children,
+    ...props
+}: HeroProps): React.ReactElement<any> => (
+    <StyledHero
+        className={classnames('anchor-hero', className)}
+        background={background}
+        align={align}
+        color={color}
+        minHeight={minHeight}
+        {...props}
+    >
+        {children}
+    </StyledHero>
+);
+
+export default Hero;
+
+// Hero.Title
+// ------------------------------------------------------------------------------------------------------------------
+
+const StyledTitle = styled(Typography)`
+    text-shadow: 0 2px 4px rgba(0, 0, 0, 0.09);
+    position: relative;
+    // https://github.com/styled-components/styled-components/issues/1816
+    && {
+        // margin: 0.375rem 0;
+    }
+`;
+
+export const HeroTitle = ({
+    className,
+    children,
+    tag = 'h1',
+    weight = 600,
+    color = 'inherit',
+    align = 'inherit',
+    size = 2,
```

Comment:
This component should be an `h1` in almost all circumstances, but designs also show it as `2rem`. `Typography` warns that setting the size is a bad idea...

Reply:
This is one of those practical over principle things IMO. For now, we'll allow tag overrides, but we should use this as a mechanism to enforce consistency with our friends on the design team.